src/CUBLAS.jl

```diff
@@ -11,8 +11,11 @@ module CUBLAS
 
 importall Base.LinAlg.BLAS
 
-using CUDArt
-using CUDArt.rt.cudaStream_t
+using CUDAdrv: OwnedPtr, CuArray, CuVector, CuMatrix
+
+CuVecOrMat{T} = Union{CuVector{T},CuMatrix{T}}
+
+const cudaStream_t = Ptr{Void}
```

Comment:
Actually, this is just a typealias that's used by the generated C wrappers, so it seems reasonable to keep it just for internal convenience.